tsdb/engine/tsm1/wal_test.go

```diff
@@ -651,6 +652,54 @@ func TestWALSegmentReader_Corrupt(t *testing.T) {
 	}
 }
 
+func TestWALRollSegment(t *testing.T) {
+	dir := MustTempDir()
+	defer os.RemoveAll(dir)
+
+	w := tsm1.NewWAL(dir)
+	if err := w.Open(); err != nil {
+		t.Fatalf("error opening WAL: %v", err)
+	}
+	const segSize = 1024
+	w.SegmentSize = segSize
+
+	values := map[string][]tsm1.Value{
+		"cpu,host=A#!~#value": []tsm1.Value{tsm1.NewValue(1, 1.0)},
+		"cpu,host=B#!~#value": []tsm1.Value{tsm1.NewValue(1, 1.0)},
+		"cpu,host=C#!~#value": []tsm1.Value{tsm1.NewValue(1, 1.0)},
+	}
+	if _, err := w.WriteMulti(values); err != nil {
+		fatal(t, "write points", err)
```

Comment:
Why not `t.Fatalf`?

Reply:
it's just following the convention of other functions in this file. fatal() wraps Fatalf